script.js

```diff
@@ -0,0 +1,103 @@
+let myLibrary = [];
+let table = document.querySelector("table");
+const newBtn = document.querySelector("#new-btn");
+const form = document.querySelector("form");
+
+function Book(title, author, pages, readStatus) {
+    this.title = title,
+    this.author = author,
+    this.pages = pages,
+    this.readStatus = readStatus;
+}
+
+Book.prototype.changeReadStatus = function() {
+    if (this.readStatus === "Read") {
+        this.readStatus = "Unread";
+    }
+    else{
+        this.readStatus = "Read";
+    }
+}
+
+const theHobbits = new Book(
```

Comment:
check if we can named value argument while creating object

Reply:
implemented the way of creating objects in which arguments are defined